examples/geojson-animation/webpack.config.js

```diff
@@ -5,6 +5,16 @@
 const resolve = require('path').resolve;
 const webpack = require('webpack');
 
+const BABEL_CONFIG = {
```

Comment:
Yeah, so this is needed because it's a safer way to load the babel configuration. The reason why the build was failing was because we had babel configuration in `package.json`. I'm not sure exactly why that was causing things to fail, but plucking it out of there and putting it here resolved the issues.